### PR TITLE
Sync content from apache

### PR DESCRIPTION
**Pull request by abesto**
This PR first merges the current `HEAD` of https://github.com/apache/incubator-zipkin-website/ into this repository, then removes all ASF-specific content, and reverts links and mentions of Apache Zipkin to OpenZipkin.

This is in preparation for the withdrawal of Zipkin from the Apache Incubator ([vote closing tomorrow](https://mail-archives.apache.org/mod_mbox/zipkin-dev/201906.mbox/%3cCAHzwyDtOyD542YJWGOzirwLrVpyc8NxE_QWqdipG66oFm_EmoQ@mail.gmail.com%3e), conclusion seems clear)

Tested by:
* `grep -Fi -e apache -e asf`
* Clicked through all the pages, they don't seem obviously broken

More (especially content) verification might be in order before merging.

### CODE REVIEW

File: generate_thrift_v1_docs/generate.sh [hunk 1]
```diff
@@ -6,7 +6,7 @@ set -x
 # Where's Waldo?
 [ ! -d ./generate_thrift_v1_docs ] && cd ..
 if [ ! -d ./generate_thrift_v1_docs ]; then
-    echo "Please run this script either from the root of the openzipkin.github.io repository"
+    echo "Please run this script either from the root of the zipkin-website repository"
```
Comment (adriancole) [inline generate_thrift_v1_docs/generate.sh]:
```suggestion
    echo "Please run this script either from the root of the openzipkin.github.io repository"
```

File: generate_thrift_v1_docs/generate.sh [hunk 2]
```diff
@@ -19,9 +19,10 @@ target_dir="${target_root}/thrift/v1"
 rm -rfv "$target_dir"
 
 # Prepare clean workspace
-cd "$(mktemp -d)"
+#   base temp dir to /tmp to avoid having to custom configure OS/x Docker
+cd "$(mktemp -d /tmp/XXXXXXXXXX)"
 git clone https://github.com/openzipkin/zipkin-api.git
-cd zipkin-api/thrift
+cd incubator-zipkin-api/thrift
```
Comment (adriancole) [inline generate_thrift_v1_docs/generate.sh]:
```suggestion
cd zipkin-api/thrift
```

File: pages/tracers_instrumentation.md [hunk 3]
```diff
@@ -32,6 +30,6 @@ the team on [Zipkin Gitter](https://gitter.im/openzipkin/zipkin/) chat.
 {: .wide-table}
 
 Did we miss a library? Please open a pull-request to
-[openzipkin.github.io](https://github.com/openzipkin/openzipkin.github.io).
+[zipkin.apache.org](https://github.com/apache/incubator-zipkin-website).
```
Comment (adriancole) [inline pages/tracers_instrumentation.md]:
```suggestion
[openzipkin.github.io](https://github.com/openzipkin/openzipkin.github.io).
```

File: _data/tracers_instrumentation.yml [hunk 4]
```diff
@@ -20,7 +20,7 @@
 
 - language: Java
   library: >-
-    [brave](https://github.com/openzipkin/brave)
+    [brave](https://github.com/apache/incubator-zipkin-brave)
```
Comment (adriancole) [inline _data/tracers_instrumentation.yml]:
```suggestion
    [brave](https://github.com/openzipkin/brave)
```

**Comment by adriancole** [comment]
ok I think this is good now! Thanks @abesto!

File: public/css/zipkin.css [hunk 5]
```diff
@@ -51,3 +51,11 @@ table.wide-table td {
 table.wide-table th {
   font-size: 0.8em;
 }
+
```
Comment (adriancole) [inline public/css/zipkin.css]: whoops missed deleting this part.. will do in a follow-up

Comment (adriancole) [reply public/css/zipkin.css]: d937dad

**Comment by adriancole** [comment]
cool! https://zipkin.io/zipkin-api/#/default still works :)

**Comment by abesto** [comment]
Great, thanks for finishing it up. I'm slightly worried about not fully understanding *why* /zipkin-api works, but for now I'll just take the win and come back to it if it breaks in the future 🤞

**Comment by adriancole** [comment]
Haha I also dont know why that works

**Pull request merged**